amgut/webserver.py

```diff
@@ -86,6 +88,10 @@ def __init__(self):
             (r"/authed/open-humans/", OpenHumansHandler),
             (r"/authed/connect/open-humans/", OpenHumansLoginHandler),
             (r"/authed/download/(.*)", DownloadHandler),
+            # (r"/authed/single/", SingleSampleHandler),
```

Comment:
It's a reminder that all the code exists for that to happen, once emperor stabilizes.